cypress/integration/tests/keycloak/create.test.ts

```diff
@@ -1,18 +1,34 @@
 import { User } from "../../models/keycloak/users/user";
 import { getRandomUserData } from "../../../utils/data_utils";
 import { UserRoles } from "../../types/constants";
+import { UserArchitect } from "../../models/keycloak/users/userArchitect";
+import { UserMigrator } from "../../models/keycloak/users/userMigrator";
 
 describe("Keycloak operations", () => {
-    it("Create user, define password, add and remove roles, remove user", () => {
+    let userArchitect = new UserArchitect(getRandomUserData());
+    let userMigrator = new UserMigrator(getRandomUserData());
+
+    before("Creating RBAC users, adding roles for them", () => {
+        User.loginKeycloakAdmin();
+        userArchitect.create();
+        userMigrator.create();
+    });
+
+    it("Login as architect and validate permissions", () => {
+        userArchitect.login();
+        userArchitect.validateRbac();
```

Comment:
validateRbac is very vast . 
Let's check for all menu one by one .
Example . validateAssessApplications(false)
validateAnalyzeApplications(false)

That way we will know exactly which test failed.

Reply:
OR

validateRbac(Assess, false)
validateRbac(Analyze, true)

Reply:
First variant is applied